CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ set(GZ_TRANSPORT_VER ${gz-transport14_VERSION_MAJOR})
 # Find gz-rendering
 
 # See CMP0072 for more details (cmake --help-policy CMP0072)
-if ((NOT ${CMAKE_VERSION} VERSION_LESS 3.11) AND (NOT OpenGL_GL_PREFERENCE))
+if (NOT OpenGL_GL_PREFERENCE)
   set(OpenGL_GL_PREFERENCE "GLVND")
 endif()
```

Comment:
Since we're setting the minimum version to 3.22.1, I think this whole section can be removed based on my understanding of https://cmake.org/cmake/help/latest/policy/CMP0072.html

Reply:
looks like it, removed the whole block in [613b6e2](https://github.com/gazebosim/gz-sensors/pull/457/commits/613b6e2018aab3bc175b0dc462683aaf3fd21e2d)